explorer/src/App.tsx

```diff
@@ -43,6 +44,20 @@ function App() {
               <TransactionDetailsPage signature={match.params.signature} />
             )}
           />
+          <Route
+            exact
+            path={"/slot/:slot"}
+            render={({ match }) => (
+              <BlockDetailsPage slot={match.params.slot} blockhash={""} />
+            )}
```

Comment:
Sorry, I changed my mind from before. I don't think we need the `/slot/:slot` path because blocks can be uniquely identified by both slot number and hash. So I think it's best to combine these under a single route path `/block/:id` where id can be either a number slot or string hash.

Reply:
Oh no worries! It's actually a very easy check. Just gotta run an `isNaN` check!
Will refactor!

Reply:
Also, agree it's better semantics wise.